src/02-room-details.js

```diff
@@ -49,8 +66,25 @@ function getRoomByDinosaurName(dinosaurs, rooms, dinosaurName) {}
       "Kit Hopkins Education Wing"
     ]
  */
-function getConnectedRoomNamesById(rooms, id) {}
+function getConnectedRoomNamesById(rooms, id) {
+  let roomObj = {}
+for (const room of rooms) {
+ roomObj[room.roomId] = room.name; 
+}
+if(!roomObj[id]){
+  return `Room with ID of '${id}' could not be found.`
+}
+let target = rooms.find(room=> room.roomId === id)
+let roomNames =[]
 
+for(let roomId of target.connectsTo){
+  if(!roomObj[roomId]){
+    return `Room with ID of 'incorrect-id' could not be found.`
```

Comment:
This is considered hardcoded. If the roomID was `abc1234`, we would still want this message with the incorrect room id in the string.